### DISCUSSION TITLE
DaB looking for  E4000  tuner

### DISCUSSION THREAD

**Discussion by yves2** [rern/rAudio#1550] [Bug Reports]
I have a dab-stick RTL2832U which was working after some adjustment
but now the program stops because the tuner is not a E4000

mediamtx 1.4.0-1
# rtl_test -t
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
Detached kernel driver
Found Rafael Micro R820T tuner
Supported gain values (29): 0.0 0.9 1.4 2.7 3.7 7.7 8.7 12.5 14.4 15.7 16.6 19.7 20.7 22.9 25.4 28.0 29.7 32.8 33.8 36.4 37.2 38.6 40.2 42.1 43.4 43.9 44.5 48.0 49.6 
[R82XX] PLL not locked!
Sampling at 2048000 S/s.
No E4000 tuner found, aborting.
Reattached kernel driver

**Comment by TranquilHegemony** [comment]
I have exactly the same problem as well. 

Build is the latest version - rAudio 20240818 - installed and have enabled the DAB Radio addon and added usbutils package but nothing more.   

 rAudio:~  #  rtl_test -t
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner
Supported gain values (29): 0.0 0.9 1.4 2.7 3.7 7.7 8.7 12.5 14.4 15.7 16.6 19.7 20.7 22.9 25.4 28.0 29.7 32.8 33.8 36.4 37.2 38.6 40.2 42.1 43.4 43.9 44.5 48.0 49.6
[R82XX] PLL not locked!
Sampling at 2048000 S/s.
No E4000 tuner found, aborting.

If I run the command 'dab-scanner-rtlsdr -F dabradio.txt' then I do get stations listed in the txt file the output is written to so the DAB Stick definitely looks to be working but no option to do this within rAudio - the DAB Radio button under the Library is just greyed out and not selectable.

**Reply by Megawaldi** [reply]
I have the same issue with a rtl-sdr blog v4 Dongle. I can scan stations, but the dab icon in library is still gray. 
Is there a solution?
